ui/app/pages/send/send-content/add-recipient/add-recipient.js

```diff
@@ -22,7 +22,7 @@ export function getToErrorObject(to, sendTokenAddress, chainId) {
   let toError = null;
   if (!to) {
     toError = REQUIRED_ERROR;
-  } else if (!isValidAddress(to)) {
+  } else if (!isValidAddress(to) && !isValidDomainName(to)) {
```

Comment:
I see we use `isValidDomainName` throughout the codebase to support ENS checks: ui/app/pages/settings/contact-list-tab/add-contact/add-contact.component.js

In this case, it has no improper fall through as part of `add-recipient.js`. It allows ENS names to move forward, and in the case of a valid domain name, not an ENS name, (ex: `brantly.com`) it will just default to the address book lookup, as domain names are fine there.